tools/terraform-bundle/config.go

```diff
@@ -5,20 +5,27 @@ import (
 	"io/ioutil"
 
 	"github.com/hashicorp/hcl"
+	"github.com/hashicorp/terraform/addrs"
+	"github.com/hashicorp/terraform/internal/getproviders"
 	"github.com/hashicorp/terraform/plugin/discovery"
 )
 
-var zeroTwelve = discovery.ConstraintStr(">= 0.12.0").MustParse()
+var zeroThirteen = discovery.ConstraintStr(">= 0.12.0").MustParse()
```

Comment:
Maybe a `FIXME` comment here would explain the temporary mismatch between the variable name and the constraint.

Reply:
Good point! The FIXME is this PR - it can't get merged until that's fixed- but I will add a comment too!